### ISSUE TITLE
Leiden+ for figures malfunctioning in DCLP

### ISSUE THREAD

**Issue by samosafuz** [papyri/xsugar#26]
In Leiden+, one normally uses `#` to generate the xml for a figure: e.g., `#seal` >> `<figure><figDesc>seal</figDesc></figure>`. Attempts to use this Leiden+ in DCLP, however, result in the **POSSIBLE ERROR** warning. It appears something is amiss with XSugar.

**Comment by samosafuz** [comment]
The malfunction is only in the direction Leiden+ >> XML. When the XML is entered manually, it generates `#` in Leiden+ as expected, and thenceforth produces no error when saving. The error, in other words, only occurs when `#` is entered in Leiden+ for an initial to conversion to XML.

**Comment by hcayless** [comment]
Looking at the grammar, I see a space is required following the `#seal `. Omitting that does lead to an error. Is it possible that's the source of the problem? Would it be better to have a beginning and ending delimiter, like `#seal#`?

**Comment by samosafuz** [comment]
Yes, you're right: I have made a mistake with the delimiting space. 

Personally, I'd prefer a delimiter on both sides (like we do with `<g>`, and especially if the `<figure>` appears midline, since it requires _two_ spaces before any subsequent text). But this is not really a priority and I'll try to remember for the next time I need to encode one of these.
